src/components/Sum.jsx

```diff
@@ -1 +1,10 @@
 // export function Sum() {}
+import React from 'react';
+
+export function Sum({ a = 0, b = 0 }) {
+  return (
+    <>
+      <p>{`Sum of ${a} and ${b} is ${a + b}`}</p>
+    </>
```

Comment:
There is unnecessary React.Fragment, just remove it
```suggestion
    <p>{`Sum of ${a} and ${b} is ${a + b}`}</p>
```